torch/optim/lr_scheduler.py

```diff
@@ -59,9 +78,26 @@ def load_state_dict(self, state_dict):
         """
         self.__dict__.update(state_dict)
 
-    def get_lr(self):
+    def get_computed_values(self):
+        # Return last computed learning rate by current scheduler
+        return self._last_computed_values
+
+    def _compute_values(self):
+        # NOTE [LR Chainable Form] Compute learning rate using chainable form of the scheduler
```

Comment:
So, the way the `Note [Foo]` convention works in the codebase, is that you have some long explanation that lives somewhere else, like:

```
Note [Foo]
~~~~~~~~~
blah blah blah
```

And then you reference the note from relevant code fragments `See Note [Foo]`. I don't see any such long explanation here. Do you want to add a longer explanation? Otherwise, just get rid of the `Note`

Reply:
Thanks for the clarification. I'll keep the comment and remove the tag as you suggest.